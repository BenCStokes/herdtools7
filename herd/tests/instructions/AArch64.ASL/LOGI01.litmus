AArch64 LOGI01
{
}
  P0               ;
 ORR W0,WZR,#3     ;
 ORR W0,W0,#4096   ;
 ORR W0,W0,#8192   ;
 AND W0,W0,#65534  ;
 EOR W0,W0,#255    ;
forall 0:X0 = 12541
 