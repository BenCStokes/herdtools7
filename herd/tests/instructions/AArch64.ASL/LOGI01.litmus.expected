Test LOGI01 Required
States 1
0:X0=12541;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=12541)
Observation LOGI01 Always 1 0
Hash=4fc2197b659a507a056299b9ecba86b6

